### PR TITLE
split images and added link to current user-stories

### LINKED ISSUE #278: Update user logos to link to corresponding user stories
The user logos are currently in a single image:

Screenshot 2023-05-04 at 6 22 39 PM

We'd like to break up this image into separate icons.  The icons that link to a user story should route to the correct page.  Logos that don't have an associated user story shouldn't link anywhere.

For example, the Scribd logo should link to [this page](https://delta.io/user-stories/scribd/). 

We should save the images in some directory that allows us to easily change the order.  There should be an intelligent filenaming convention.  e.g. scribd.png should link to user-stories/scribd.

### PR DESCRIPTION
**Pull request by dev-smart-ui**
resolves #278

### CODE REVIEW

**Comment by dennyglee** [comment]
Thanks @dev-smart-ui - this is looking pretty good.  Saying this, could you add some more vertical spacing between the different logos.  The horizontal spacing is quite good but the vertical spacing seems squat.

As well, could you include all of the logos from the original diagram as well as any new ones?  Thanks!

**Comment by dev-smart-ui** [comment]
Hey @dennyglee ! 
Thanks for your feddback. 

I added more row spacing between the icons. As for quontity of icons I added all of them from the current scheme (included Kubit), but I changed the grid from 6(columns) x 4(rows) to the 5(columns) x 5(rows). In other way we will get only one icons for last row. It looks not well. 

What do you think?

File: src/components/pages/community/ContributeToDeltaSection/contributors.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import disney from "./img/disney.png";
+import grafana from "./img/grafana.png";
+import safeGraph from "./img/safeGraph.png";
+import starBust from "./img/starBurst.png";
+import td from "./img/td.png";
+import talend from "./img/talend.png";
+import twilio from "./img/twilio.png";
+import ucDavis from "./img/ucDavis.png";
+
+export const contributors = [
```
Comment (dennyglee) [inline src/components/pages/community/ContributeToDeltaSection/contributors.js:27]: Could we rename this to `orgContributors` vs `contributors` so we don't confuse between organizations vs. contributors (maintainers).

File: src/components/pages/community/ContributeToDeltaSection/index.jsx [hunk 2]
```diff
@@ -28,10 +45,18 @@ const ContributeToDeltaSection = () => (
     centeredHeader
     padding="xxxl"
   >
-    <Grid columns={{ md: 1 }} gutter="xxl">
-      <ContributeToDeltaColumn>
-        <img src={organizationsImg} alt="" />
-      </ContributeToDeltaColumn>
+    <Grid columns={{ xs: 2, sm: 3, md: 4, lg: 5 }} gutter="lg" rowGutter="xl">
```
Comment (dennyglee) [inline src/components/pages/community/ContributeToDeltaSection/index.jsx:48]: Can we try rowGutter = "xxl"?

**Review by dennyglee — CHANGES_REQUESTED** [review]
Two quick changes - thanks!

**Comment by dennyglee** [comment]
Thanks @dev-smart-ui  - can we try xxl row gutter just to see if it looks better?  As for the icons - that's fair, just wanted to confirm that I can add another 5 icons :)

File: src/components/pages/community/ContributeToDeltaSection/contributors.js (hunk 1 above)
Comment (dennyglee) [reply src/components/pages/community/ContributeToDeltaSection/contributors.js]: thanks!

File: src/components/pages/community/ContributeToDeltaSection/index.jsx (hunk 2 above)
Comment (dennyglee) [reply src/components/pages/community/ContributeToDeltaSection/index.jsx]: thanks, looks great!

**Review by dennyglee — APPROVED** [review]
awesome, thanks!

**Review by dennyglee — APPROVED** [review]
awesome, thanks!

**Pull request merged**